features/kinderen.feature

```diff
@@ -5,78 +5,121 @@ Functionaliteit: Kinderen van een persoon raadplegen
   Van een persoon worden -indien gevraagd met de fields parameter- de kinderen geleverd.
   Dit bevat enkele identificerende eigenschappen van de kinderen.
 
-  @gba
   Rule: de actuele gegevens van kinderen worden geleverd
 
     @gba
     Scenario: de naam van een kind is gecorrigeerd
-      Gegeven de persoon met burgerservicenummer 999996150 heeft de volgende kinderen in de registratie
-      | Categorie | Voornamen (02.10) | Voorvoegsel (02.30) | Geslachtsnaam (02.40) | Onjuist (84.10) |
-      | 9         | William           |                     | Postma                |                 |
-      | 59        | William           | de                  | Vries                 |                 |
-      | 59        | William           | de                  | Boer                  | O               |
-      | 9         | Sebastiaan        | de                  | Boer                  |                 |
-      | 9         | Walter            | de                  | Boer                  |                 |
-      | 59        | Walter            |                     | Messeritz             | O               |
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999996150 |
+      En de persoon heeft een 'kind' met de volgende 'naam' gegevens
+      | naam                  | waarde  |
+      | voornamen (02.10)     | William |
+      | geslachtsnaam (02.40) | Postma  |
+      En het 'kind' heeft de volgende 'naam' historie gegevens
```

Comment:
Is uit dit scenario wel voldoende af te leiden op welk kind de diverse historie gegevens betrekking hebben? In develop/features/README.md staat n.m.m. niets waaruit je kunt afleiden hoe dit scenario te interpreteren. Zelf kan ik denk ik wel de logica ontdekken maar moet niet ergens vastgelegd worden welke logica, in de programmatuur die dit bestand verwerkt, is verwerkt?